tests/test-metaboxes.php

```diff
@@ -29,8 +29,6 @@ public function test_update_font_stacks() {
 		$this->assertArrayHasKey( 'part', $wp_meta_boxes );
 		$this->assertArrayHasKey( 'metadata', $c->metadata );
 		$this->assertArrayHasKey( 'general-book-information', $c->metadata['metadata'] );
-		$this->assertArrayHasKey( 'additional-catalogue-information', $c->metadata['metadata'] );
```

Comment:
@connerbw All of these requests have been addressed—want to take one last look before I merge?